public/celestia-app.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+# ASCII art
+echo "          __        __  _                       "
+echo " _______ / /__ ___ / /_(_)__ ________ ____  ___ "
+echo "/ __/ -_) / -_|_-</ __/ / _ \`/___/ _ \`/ _ \/ _ \\"
+echo "\\__/\\__/_/\\__/___/\\__/_/\\_,_/    \\_,_/ .__/ .__/"
+echo "                                    /_/  /_/    "
+
+# Rest of your script...
+echo ""
+
+# Create a log file to capture detailed logs
+LOGFILE="$HOME/celestia-temp/logfile.log"
+TEMP_DIR="$HOME/celestia-temp"
+
+# Check if the directory exists
+if [ -d "$TEMP_DIR" ]; then
+    read -p "Directory $TEMP_DIR exists. Do you want to clear it out? (y/n) " -n 1 -r
+    echo    # move to a new line
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+        rm -rf "$TEMP_DIR"
+        echo "Directory $TEMP_DIR has been removed."
+    fi
+fi
+
+mkdir -p "$TEMP_DIR"
+touch "$LOGFILE"
+
+# Log and print the log file location
+echo "Log file is located at: $LOGFILE" | tee -a "$LOGFILE"
+
+# Create a temporary directory to work from
+TEMP_DIR="$HOME/celestia-temp"
```

Comment:
`TEMP_DIR` appears defined twice so this line may be safe to remove.

Reply:
7ba84ebc77612d7e7ecc0486acdb412f57102225